operator/src/test/java/org/keycloak/operator/testsuite/integration/WatchedSecretsTest.java

```diff
@@ -78,6 +80,47 @@ public void testSecretsAreWatched() {
         });
     }
 
+    @Test
+    public void testClonedSecretUnmodified() {
+        Log.info("Creating new Keycloak CR example");
+        var kc = getTestKeycloakDeployment(true);
+        deployKeycloak(k8sclient, kc, true);
+
+        AtomicInteger createCount = new AtomicInteger();
+        AtomicInteger updateCount = new AtomicInteger();
+
+        k8sclient.secrets().withName("example-tls-secret").inform(new ResourceEventHandler<Secret>() {
```

Comment:
I might be missing something but should not we rather be informed on the cloned secret to see if the Operator touches it or not?

Reply:
Ugh, that's what I meant to track, I'll update this.

Reply:
This has been reimplemented without labels.  I can probably also take it a bit further to use a cache to scan for keycloaks.